src/Apps/Auction/Components/RegistrationForm.tsx

```diff
@@ -186,17 +194,54 @@ const validationSchema = Yup.object().shape({
   ),
 })
 
+type TrackErrors = (errors: string[]) => void
```

Comment:
We pass our form submission error handler in from above where tracking is set up. This is its proptype.